src/screens/Home/surveyList.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef } from 'react';
+
+import Slider from 'react-slick';
+
+import testSurvey from 'images/auth_background.png';
+import slickSettings from 'screens/Home/slickSettings';
+
+const SurveyList = () => {
+  const sliderRef = useRef(null);
+
+  useEffect(() => {
+    window.addEventListener('wheel', (e) => {
+      slideList(e.wheelDelta);
+    })
+  }, []);
+
+  const slideList = (wheel) => {
+    wheel > 0 ? (
+      sliderRef.current.slickNext()
+    ) : (
+      sliderRef.current.slickPrev()
+    )
+  }
+
+  return (
+    <div className="survey-list">
+      <Slider { ...slickSettings } ref={ sliderRef }>
+        <div>
+          <img src={ testSurvey } alt="test" className="survey-list__image" />
+        </div>
+        <div>
+          <img src={ testSurvey } alt="test" className="survey-list__image" />
+        </div>
+        <div>
+          <img src={ testSurvey } alt="test" className="survey-list__image" />
+        </div>
+      </Slider>
+
+      <h3 className="survey-list__title">Working from home Check-In</h3>
+      <p className="survey-list__description">
+        We would like to know about how you feel about our work from home.
+      </p>
+
+      <div className="survey-list__navigation">
+        <button className="button button--circle button--circle-right">
```

Comment:
@rafayet-monon what do you think if we design a structure like this?
```html
<div>
  <div>
    <h3 />
    <p />
  </div>

  <div>
    <button />
  </div>
</div>
```
Then use flex to layout them.
So we can omit this `button--circle-right`
Something like this https://jsfiddle.net/lahphim/bsfpw7L0/

Reply:
@Lahphim fixed in [9b8b432](https://github.com/rafayet-monon/surveyor/pull/16/commits/9b8b432e092af9f7a330f774af2434887167c89b).